superset/models/core.py

```diff
@@ -885,36 +885,67 @@ def wrapper(*args, **kwargs):
             if g.user:
                 user_id = g.user.get_id()
             d = request.form.to_dict() or {}
+
             # request parameters can overwrite post body
             request_params = request.args.to_dict()
             d.update(request_params)
             d.update(kwargs)
+
             slice_id = d.get('slice_id')
+            dashboard_id = d.get('dashboard_id')
 
             try:
                 slice_id = int(
                     slice_id or json.loads(d.get('form_data')).get('slice_id'))
             except (ValueError, TypeError):
                 slice_id = 0
 
+            stats_logger.incr(f.__name__)
+            value = f(*args, **kwargs)
+            sesh = db.session()
+
+            logs = []
+            duration_ms = (datetime.now() - start_dttm).total_seconds() * 1000
```

Comment:
Nit. I think the duration should really only capture the `f(*args, **kwargs)` call.